apps/typescriptcourse/src/server/routers/resources.ts

```diff
@@ -0,0 +1,19 @@
+import {publicProcedure, router} from '@skillrecordings/skill-lesson'
+import {z} from 'zod'
+import {getLesson} from '../../lib/lesson'
+
+export const lessonResourcesRouter = router({
+  byExerciseSlug: publicProcedure
+    .input(
+      z.object({
+        slug: z.string(),
+      }),
+    )
+    .query(async ({ctx, input}) => {
+      const lesson = await getLesson(input.slug)
+
+      const github = lesson.github
+
+      return {github}
+    }),
+})
```

Comment:
grab the github resource when needed. this can include additional resources for this product in the future.